Makefile

```diff
@@ -0,0 +1,54 @@
+KERNEL_OUTPUT=$(CURDIR)/output/vmlinux.container
+IMAGE_OUTPUT=$(CURDIR)/output/container.img
+
+ifdef http_proxy
+PROXY=--build-arg http_proxy=$(http_proxy)
+RUN_PROXY=--env http_proxy=$(http_proxy)
+endif
+
+ifdef https_proxy
+PROXY+=--build-arg https_proxy=$(https_proxy)
+RUN_PROXY+=--env https_proxy=$(https_proxy)
+endif
+
+IMG_SIZE=128M
+
+image: docker_build output
+	docker run \
+		--runtime runc \
+		--privileged \
+		-v /dev:/dev \
+		$(RUN_PROXY) \
+		-ti \
+		-v $(CURDIR)/image_creator.sh:/image_creator.sh \
```

Comment:
We could copy that one from the Dockerfile directly and make that command line a bit simpler.